src/Finch_Run.hpp

```diff
@@ -110,9 +110,13 @@ class Layer
 
     auto writeSolidificationData() { return solidification_data_.write(); }
 
-    auto getSolidificationDataBounds()
+    std::array<double, 3> getLowerSolidificationDataBounds()
```

Comment:
So you don't like auto

Reply:
I've warmed up to auto but for variables that are passed between Finch and ExaCA I feel a lot better about having an explicit type where possible given to avoid any potential incompatibility